plugins/serializers/extr/README.md

```diff
@@ -9,26 +9,29 @@ The `extr` output data format converts metrics into JSON documents, performing t
         --> "usage":{"avg":50,"max":100,"min":1}
         ifAdminStatus_old="Up",ifAdminStatus_new="Down"
         --> "ifAdminStatus":{"old":"Up","new":"Down"}
+        *You could also use old_ifAdminStatus="Up",new_ifAdminStatus="Down" to achieve the same result.
 
-- Groups metric fields appended with _key.
+- Groups metric fields appended with _key to a keys group
         ifIndex_key=1, name_key="1:2"
-        --> "key":{ifIndex:1, name:"1:2"}
+        --> "keys":{ifIndex:1, name:"1:2"}
+        *You could also use ifIndex_keys=1,name_keys="1:2"
 
-- Groups metric fields appended with _tag.
+- Groups metric fields appended with _tag to a tags group
         partNumber_tag="1647G-00129 800751-00-01", revision_tag="01"
         --> "tags":{partNumber:"1647G-00129 800751-00-01", revision:"01"}
+        *You could also use partNumber_tags="1647G-00129 800751-00-01", revision_tags="01"
 
 - Groups like metric names into a toplevel map. Name of group is same as name, but with first char lowercase
         "fanStats" :[{grouped_FanStats_Metric1}, {grouped_FanStats_Metric2} ]
 
 - Creates nested JSON schema by parsing underscore "_" seperated field keys
-        cpu1_subcore_core_keys=2
-        cpu2_subcore_core_keys=5
+        cpu1_subcore_core_key=2
```

Comment:
Are we planning to change the CPU utilization message? Wondering what this change is - "keys to key"

Reply:
No changes.  Right now any fields ending with _key (i.e. x1_key, x2_key)  is grouped into {keys: { x1, x2 }} grouping.  Same goes for any field with _tag which generates {tags:{x1, x2}}.  Thus, _key and _tag suffix are "special" suffixes in that key becomes keys and tag becomes tags.

That said, when I added support for nested JSON you could "also" just end field names with _keys, which would also create the same JSON format. (i.e. keys : { ...}).